pkg/registry/helpers.go

```diff
@@ -1,8 +1,6 @@
 package registry
 
-import (
-	"github.com/google/go-containerregistry/pkg/v1"
-)
+import v1 "github.com/google/go-containerregistry/pkg/v1"
```

Comment:
same thing... I feel like I am missing some idiomatic Go stuff. would love to learn more (to avoid this)